packages/grafana-data/src/utils/OptionsUIBuilders.ts

```diff
@@ -137,6 +179,11 @@ export class PanelOptionsEditorBuilder<TOptions> extends OptionsUIRegistryBuilde
   StandardEditorProps,
   PanelOptionsEditorItem<TOptions>
 > {
+  addNestedBuilder<Sub>(opts: NestedPanelOptions<Sub>) {
```

Comment:
nit, to avoid leaking the `builder` implementation detail:

```suggestion
  addNestedOptions<Sub>(opts: NestedPanelOptions<Sub>) {
```